src/robusta/core/reporting/blocks.py

```diff
@@ -76,6 +76,31 @@ def __init__(self, filename: str, contents: bytes):
         """
         super().__init__(filename=filename, contents=contents)
 
+    def truncate_content(self,  max_file_size_bytes: int) -> bytes:
+        """
+        Truncates the log file by removing lines from the beginning until its size is within the given limit.
+        """
+        content_length = len(self.contents)
```

Comment:
this is only correct for utf-8 strings
you should either convert it to utf-8 before you calculate the content_length and return the converted value or you should use this calculation for length here
`content_length = sys.getsizeof(self.contents) - sys.getsizeof("")`
(just double check to be sure)
below here. its okay to use len because it is utf-8 encoded